wherehows-web/app/components/datasets/containers/dataset-health.ts

```diff
@@ -69,6 +69,22 @@ export default class DatasetHealthContainer extends Component {
    */
   tableData: Array<IHealthScore> = [];
 
+  /**
+   * Passed in from the higher level component, we use this property in order to determine whether the dataset health
+   * tab is the currently selected tab
+   * @type {string}
+   */
+  tabSelected: string;
+
+  /**
+   * Calculated from the currently selected tab to determine whether this container is the currently selected tab.
+   * Note: Highcharts calculates size and other chart details upon initial render and doesn't do a good job of handling
+   * rerenders. Because of this we want those calculations to take place when dataset health is the currently active tab,
+   * otherwise we will insert elements off screen and size will default to 0 and we lose our charts
+   * @type {ComputedProperty<boolean>}
+   */
+  isActiveTab = equal('tabSelected', 'health');
```

Comment:
Let's use the equal decorator here instead, this will also help with the deprecation of defineProperty, since I assume it'll be fixed in ember-decorators and propagate through the app
```ts
@equal('tabSelected', Tabs.Health)
isActiveTab: boolean;
```